internal/integration/api/reboot.go

```diff
@@ -90,7 +90,7 @@ func (suite *RebootSuite) TestRebootNodeByNode() {
 			if suite.Cluster != nil {
 				// without cluster state we can't do deep checks, but basic reboot test still works
 				// NB: using `ctx` here to have client talking to init node by default
-				suite.AssertClusterHealthy(ctx)
+				suite.AssertClusterHealthy(suite.ctx)
```

Comment:
this drops health assertion out of `ctx` timeout, but probably that's fine

Reply:
moved back to ctx. with the increased timeout it should be fine anyways.